Package.swift

```diff
@@ -76,7 +80,8 @@ let package = Package(
         
         .testTarget(
             name: "TSCBasicTests",
-            dependencies: ["TSCTestSupport", "TSCclibc"]),
+            dependencies: ["TSCTestSupport", "TSCclibc"],
+            exclude: ["Inputs", "processInputs"]),
```

Comment:
This works just fine but I'm not sure if it's perhaps too broad and should be `Inputs/` etc?